controllers/openshift_controller.go

```diff
@@ -1727,14 +1742,119 @@ func (r *KataConfigOpenShiftReconciler) enablePeerPods() error {
 		},
 	}
 
-	err = r.Client.Create(context.TODO(), &peerpodconf)
+	err := r.Client.Create(context.TODO(), &peerpodconf)
 	if k8serrors.IsAlreadyExists(err) {
 		return nil
 	} else if err != nil {
 		return err
 	}
 
-	err = r.createRuntimeClass("kata-remote-cc", "0.25", "350Mi")
+	//Create MachineConfig for kata-remote hyp CRIO config
+	err = r.createMcFromFile(peerpodsCrioMachineConfigYaml)
+	if err != nil {
+		r.Log.Error(err, "Unable to create MachineConfig ", "peerpodsCrioMachineConfig", peerpodsCrioMachineConfig)
+		return err
+	}
+
+	//Create MachineConfig for kata-remote hyp config toml
+	err = r.createMcFromFile(peerpodsKataRemoteMachineConfigYaml)
+	if err != nil {
+		r.Log.Error(err, "Unable to create MachineConfig ", "peerpodsKataRemoteMachineConfig", peerpodsKataRemoteMachineConfig)
+		return err
+	}
+
+	// Create runtimeClass config for peer-pods
+	err = r.createRuntimeClass(peerpodsRuntimeClassName, peerpodsRuntimeClassCpuOverhead, peerpodsRuntimeClassMemOverhead)
+	if err != nil {
+		r.Log.Error(err, "Unable to create RuntimeClass ", "peerpodsRuntimeClassName", peerpodsRuntimeClassName)
+		return err
+	}
+	return nil
+}
+
+func (r *KataConfigOpenShiftReconciler) disablePeerPods() error {
+	peerpodconf := v1alpha1.PeerPodConfig{
+		TypeMeta: metav1.TypeMeta{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "peerpodconfig-example",
```

Comment:
Just a nitpick but we might want to use another name in published code than "-example"...

Reply:
Sure, I'll change it